pages/api/programs/index.ts

```diff
@@ -29,20 +29,22 @@ import { getNewsHandler, postNewsHandler } from '../../../lib/api/newsLogic';
  *                       type: dateTime
  *                       example: Jan-08-2022
  *   post:
- *     description: Creates valid news headline
+ *     description: Creates valid programs headline
  *     parameters:
  *       - name: title
  *         required: true
- *         description: The title of the news headline.
+ *         description: The title of the programs headline.
  *         schema:
  *              type : string
  *       - name: content
  *         required: true
- *         description: The content of the news headline.
+ *         description: The content of the programs headline.
  *         schema:
  *              type : string
```

Comment:
🔧 Swagger docs are outdated, they do not contain the new fields (image and date)